core/src/main/java/edu/ucr/cs/riple/core/metadata/field/FieldInitializationAnalysis.java

```diff
@@ -71,6 +71,7 @@ public Stream<OnMethod> findInitializers(Set<OnField> uninitializedFields) {
   }
 
   /** Stores class field / method initialization status. */
+  @SuppressWarnings("JavaLangClash")
```

Comment:
Please solve in a follow up diff. One option is to use `Klass` or `ClassRecord`

Reply:
Sure, `ClassRecord` sounds really good to me.